### PR TITLE
added relationselector and severityvaluesgroup component

### PR DESCRIPTION
**Pull request by saberlynx**
- added a component "RelationSelector" to select between '=', '<' and '>'. All it does is select relation.
- added a component "SeverityValuesGroup" replacing CvssBaseGroup (it is now more general) that has "RelationSelector" as child, enabling user to do eg. "cvss_base>3"

**Checklist**:

- [x] Tests
- [x] [CHANGELOG](https://github.com/greenbone/gsa/blob/master/CHANGELOG.md) Entry

### CODE REVIEW

File: gsa/src/web/components/powerfilter/relationselector.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+/* eslint-disable react/prop-types */
+import React from 'react';
+import Select from 'web/components/form/select';
+import PropTypes from 'web/utils/proptypes.js';
+
+const RelationSelector = ({relation, onChange}) => (
+  <div>
```
Comment (bjoernricks) [inline gsa/src/web/components/powerfilter/relationselector.js]: I suppose the extra div shouldn't be necessary. Just return the Select component.

File: gsa/src/web/components/powerfilter/relationselector.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    <Select
+      value={relation}
+      onChange={onChange}
+      items={[
+        {label: 'is equal to', value: '='},
+        {label: 'is greater than', value: '>'},
```
Comment (bjoernricks) [inline gsa/src/web/components/powerfilter/relationselector.js]:
```suggestion
        {label: _('is greater than'), value: '>'},
```

File: gsa/src/web/components/powerfilter/relationselector.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        {label: 'is less than', value: '<'},
```
Comment (bjoernricks) [inline gsa/src/web/components/powerfilter/relationselector.js]:
```suggestion
        {label: _('is less than'), value: '<'},
```

File: gsa/src/web/components/powerfilter/severityvaluesgroup.js [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+  }
+
+  render() {
+    const severity = parseSeverity(this.props.filter.get(this.props.name));
+    const newRelation = this.state.relation;
+    const keyword = this.props.name;
+    return (
+      <div>
+        <FormGroup title={this.props.title}>
+          <RelationSelector
```
Comment (bjoernricks) [inline gsa/src/web/components/powerfilter/severityvaluesgroup.js]: Would be nice to add an additional <Divider> around RelationSelector and NumberField here to add some margin between them.
<Divider><ReleationSelector .../><NumberField .../></Divider>

File: gsa/src/web/components/powerfilter/relationselector.js (hunk 2 above)
Comment (saberlynx) [reply gsa/src/web/components/powerfilter/relationselector.js]: in the future should all labels/titles have this format?

**Pull request merged**